src/amo/components/HomepageShelves/index.js

```diff
@@ -28,6 +31,12 @@ type InternalProps = {|
 
 export const HOMESHELVES_ENDPOINT_COLLECTIONS = 'collections';
 export const HOMESHELVES_ENDPOINT_SEARCH = 'search';
+export const HOMESHELVES_ENDPOINT_RANDOM_TAG = 'random-tag';
+
+function getTagFromUrl(api_url: string): string {
```

Comment:
CamelCase please 😃

Reply:
Also, I think we generally prefer using `const` and arrow functions for defining functions. E.g., `const getTagFromUrl = (apiUrl: string): string => {`...

Reply:
it is camelCase already 😄 just like every other `function xxxYyyyZzzz` I could find in addons-frontend.  Can you point to an example of what you're talking about?

Reply:
Sorry, I should have been more specific: `api_url`.